docs/config.toml

```diff
@@ -39,6 +39,14 @@ imageCrunchyPGAdmin = "registry.developers.crunchydata.com/crunchydata/crunchy-p
 repository = "registry.developers.crunchydata.com/crunchydata"
 repositoryPrivate = "registry.developers.com/crunchydata"
 postgresOperatorTag = "ubi8-5.1.0-0"
+PGBouncerComponentTagUbi8 = "ubi8-1.16-1"
+PGBouncerComponentTagCentos8 = "centos8-1.16-1"
+PGBouncerTagUbi8 = "ubi8-5.1.0-0"
+PGBouncerTagCentos8 = "centos8-5.1.0-0"
+postgres14GIS31ComponentTagUbi8 = "ubi8-14.1-3.1-1"
+postgres14GIS31ComponentTagCentos8 = "centos8-14.1-3.1-1"
+postgres14GIS31TagUbi8 = "ubi8-14.2-3.1-5.1.0-0"
+postgres14GIS31TagCentos8 = "centos8-13.6-3.1-5.1.0-0"
```

Comment:
:wrench: This should be for PG 14 I think

Reply:
updated